src/Directory.Build.props

```diff
@@ -56,11 +56,14 @@
   <ItemGroup>
     <!-- Profiler library items for all of the native platforms. -->
     <MonitorProfilerLibraryFile Include="@(NativeArtifactDirectories->'%(Identity)\%(LibraryPrefix)MonitorProfiler%(LibraryExtension)')" />
+    <MutatingMonitorProfilerLibraryFile Include="@(NativeArtifactDirectories->'%(Identity)\%(LibraryPrefix)MutatingMonitorProfiler%(LibraryExtension)')" />
   </ItemGroup>
 
   <ItemGroup>
     <!-- Profiler symbols items for all of the native platforms. -->
     <MonitorProfilerSymbolsFile Include="@(NativeArtifactDirectories->'%(Identity)\%(LibraryPrefix)MonitorProfiler%(SymbolsExtension)')" />
+    <MutatingMonitorProfilerSymbolsFile Include="@(NativeArtifactDirectories->'%(Identity)\%(LibraryPrefix)MutatingMonitorProfiler%(SymbolsExtension)')" />
+    <CommonMonitorProfilerSymbolsFile Include="@(NativeArtifactDirectories->'%(Identity)\%(LibraryPrefix)CommonMonitorProfiler%(SymbolsExtension)')" />
```

Comment:
Are the symbols for the common static library not included in the symbols for the shared libraries?